README.md

```diff
@@ -12,6 +12,7 @@
 
 [![Gitter](https://badges.gitter.im/iot-event-analytics/community.svg)](https://gitter.im/iot-event-analytics/community)
 [![License](https://img.shields.io/badge/License-MPL%202.0-blue.svg)](https://opensource.org/licenses/MPL-2.0)
+[![C++ SDK](https://github.com/boschglobal/iot-event-analytics/actions/workflows/build_cpp_sdk_cmake.yml/badge.svg?branch=feature%2Fgithub_actions_build_cpp_sdk)](https://github.com/boschglobal/iot-event-analytics/actions/workflows/build_cpp_sdk_cmake.yml)
```

Comment:
I think this should point to the upstream repo. (genivi)

Reply:
I think so too :)